modules/mod_menu/src/Helper/MenuHelper.php

```diff
@@ -156,13 +156,16 @@ public static function getList(&$params)
                         $item->flink = Route::_($item->flink);
                     }
 
+                    // rel can now have multiple values, combine the array into a space-separated string
```

Comment:
```suggestion
                    // rel can have multiple values, combine the array into a space-separated string
```

Just pedantry. in 2 years it's not "now" any more.